### PR TITLE
Update xfce4-notifyd specific gtk3 style. Related to #677.

### PR DESCRIPTION
**Pull request by ragazenta**
| Before | After |
| --- | --- |
| xfce-notifyd-before | xfce-notifyd-after |

Gtk3 styles for xfce4-notifyd had been moved to scss in https://github.com/numixproject/numix-gtk-theme/commit/30b14f77d05389d80b395720e6603836a579edd4. But some styles is not really match to osd styles (border, border-radius) if using Default theme. I found [Default/gtk.css](https://git.xfce.org/apps/xfce4-notifyd/tree/themes/gtk-3.20/Default/gtk.css?h=xfce4-notifyd-0.4.2) defines some _styles_ with low priority that will be used if that _style_ isn't defined in main gtk theme.

I conclude that `#XfceNotifyWindow` selector isn't defined properly. It should be like this [guide](http://docs.xfce.org/apps/notifyd/theming).

I only define styles based on Default/gtk.css, just copying osd styles.

Related to #677.

### CODE REVIEW

File: src/gtk-3.20/scss/widgets/_osd.scss [hunk 1]
```diff
@@ -23,8 +23,7 @@
         }
     }
 
-    button.osd,
-    #XfceNotifyWindow button {
```
Comment (khurshid-alam) [inline src/gtk-3.20/scss/widgets/_osd.scss:27]: @ragazenta 

Why did you remove button style on osd ? Does it inherit the button style from parent ? Show some more screenshot how will it work without it?

**Comment by khurshid-alam** [comment]
@ragazenta ?

File: src/gtk-3.20/scss/widgets/_osd.scss (hunk 1 above)
Comment (ragazenta) [reply src/gtk-3.20/scss/widgets/_osd.scss:27]: Yes, it inherit the button style from main GTK themes.

**Comment by khurshid-alam** [comment]
@ragazenta 

Thanks.
Can you move your code to `src/gtk-3.20/scss/apps/_xfce.scss`? I think that way all xfce code remain at one place. Please test it works after you move.

Thanks.

**Comment by ragazenta** [comment]
@khurshid-alam 

Done moving code. It works as expected.
